<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataBase</class>
 <widget class="QMainWindow" name="DataBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>601</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1067</width>
    <height>601</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1067</width>
    <height>601</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DataBase</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1067</width>
     <height>555</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1067</width>
     <height>555</height>
    </size>
   </property>
   <widget class="QPushButton" name="open_DB_button">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 190, 111);</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>851</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_db_name">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="table_comboBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>291</width>
      <height>26</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 163, 72);</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="bd_name_label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>291</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="text">
     <string>Input database name (historical_figures)</string>
    </property>
   </widget>
   <widget class="QLabel" name="table_name_label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>311</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="text">
     <string>Choose table (h_f)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="del_id">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>440</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="del">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>420</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="text">
     <string>Deleting</string>
    </property>
   </widget>
   <widget class="QLabel" name="del_label">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>420</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Input id</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>20</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="del_but">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>440</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(255, 120, 0);</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="add">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="text">
     <string>Adding</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_id">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>50</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_birthday">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>100</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_deathday">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>100</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_rating">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>50</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="add_but">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>280</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 190, 111);</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_label0">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>30</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_label1">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>80</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="text">
     <string>birthday</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_label2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>80</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="text">
     <string>deathday</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_label3">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="text">
     <string>rating</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_label4">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>130</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_name">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>150</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="add_label5">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>180</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="text">
     <string>position</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_position">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>200</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="add_label6">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>230</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="text">
     <string>contribution</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_contribution">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>250</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="management">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="text">
     <string>Control</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>0</y>
      <width>21</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>450</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="filt_ratingm5">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>330</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 190, 111);</string>
    </property>
    <property name="text">
     <string>rating &gt; 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filt_ratingl5">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>360</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 163, 72);</string>
    </property>
    <property name="text">
     <string>rating &lt; 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="filt">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>310</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="text">
     <string>Filtration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filt_stop">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>390</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 120, 0);</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sort_stop">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>520</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 97, 0);</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sort_ASC">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>500</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/arrow_up.png);
background-color: rgb(198, 70, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="sort_DESC">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>500</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/arrow_down.png);
background-color: rgb(198, 70, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sort">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>470</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="text">
     <string>Sorting</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sort_comboBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>490</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 120, 0);</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>id</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>id</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>имя</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>год рождения</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>год смерти</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>достижение</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>рейтинг</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>должность</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>400</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>530</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>290</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>line_4</zorder>
   <zorder>line_6</zorder>
   <zorder>line_7</zorder>
   <zorder>line_5</zorder>
   <zorder>open_DB_button</zorder>
   <zorder>tableView</zorder>
   <zorder>input_db_name</zorder>
   <zorder>table_comboBox</zorder>
   <zorder>bd_name_label</zorder>
   <zorder>table_name_label</zorder>
   <zorder>del_id</zorder>
   <zorder>del</zorder>
   <zorder>del_label</zorder>
   <zorder>line</zorder>
   <zorder>del_but</zorder>
   <zorder>add</zorder>
   <zorder>add_id</zorder>
   <zorder>add_birthday</zorder>
   <zorder>add_deathday</zorder>
   <zorder>add_rating</zorder>
   <zorder>add_but</zorder>
   <zorder>add_label0</zorder>
   <zorder>add_label1</zorder>
   <zorder>add_label2</zorder>
   <zorder>add_label3</zorder>
   <zorder>add_label4</zorder>
   <zorder>add_name</zorder>
   <zorder>add_label5</zorder>
   <zorder>add_position</zorder>
   <zorder>add_label6</zorder>
   <zorder>add_contribution</zorder>
   <zorder>management</zorder>
   <zorder>line_3</zorder>
   <zorder>filt_ratingm5</zorder>
   <zorder>filt_ratingl5</zorder>
   <zorder>filt</zorder>
   <zorder>filt_stop</zorder>
   <zorder>sort_stop</zorder>
   <zorder>sort_ASC</zorder>
   <zorder>sort_DESC</zorder>
   <zorder>sort</zorder>
   <zorder>sort_comboBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>